docs/start/integration.md

```diff
@@ -5,111 +5,119 @@ title: Integration Guide
 
 ## Node Management​
 
+You can choose to either run a **full node** or a **smeshing node**. The former is a pre-requisite for the latter, and the difference between the two node types is that the full node only stores the blockchain state and executes the transactions in valid blocks, while the smeshing node performs smeshing as well.
 
-### Running a Private Node
+### Running a Full Node
 
-See [[Running a Node|Smesher Guide#running-a-node]] in the [[Smesher Guide]]. Obviously a node backing an application does not need to smesh (mine), so you can ignore the steps related to PoST initialization and mining. We strongly encourage you to run multiple, redundant nodes so that you have a fallback in case one node has issues or falls out of sync. We also encourage you to take regular snapshots of the data directory of your nodes (ideally, after closing the node) so that you can restore your node more quickly in case of a problem, rather than needing to sync from scratch, which is time-consuming.
+To start running a full node, first ensure that your computer meets (or better yet, exceeds) the minimum [system requirements](./smeshing/requirements.md). Then, you can follow the "running a local Full Node" section of the:
+
+- [Basic Smeshing](./smeshing/smeshing_basic/smeshing.md/#running-a-local-full-node) guide to easily set up your node using Smapp (GUI), or the
+- [Advanced Smeshing](./smeshing/smeshing_adv/setup.md/#running-a-local-full-node) guide to set up your full node using the `go-spacemesh` (CLI).
+
+If you only need to run a Full Node for integration purposes, you can ignore the instructions for smeshing.
+
+We encourage you to run multiple, redundant nodes so that you have a fallback in case one node has issues or falls out of sync. We also encourage you to take regular snapshots of the data directory of your nodes (ideally, after closing the node) so that you can restore your node more quickly in case of a problem, rather than needing to sync from scratch, which is time-consuming.
 
 ### Connecting to a Public Node
 
-We don't currently operate public nodes, but we intend to in the future. This depends upon improving the existing node API. You may follow [this issue](https://github.com/spacemeshos/pm/issues/250) for updates. For now, if you plan to build an application on Spacemesh, you'll need to rely on running your own nodes.
+Currently, we do not operate public nodes. However, we do plan to do so in the future. This depends upon improving the existing node API. You may follow [this issue](https://github.com/spacemeshos/pm/issues/250) for updates. For now, if you plan to build an application on Spacemesh, you will need to run your own nodes.
```

Comment:
I think we can repeat here the v2alpha API info.

Reply:
Which info exactly? This one?

"We're currently working on the new version of the API. It's currently available as v2alpha but we strongly recommend doing any new integrations with it. The old version v1 will be deprecated afterwards. The API description is available [here](https://github.com/spacemeshos/api/tree/master/spacemesh/v2alpha1). There is also public instance exposing the JSON-RPC api at mainnet-api.spacemesh.network"

Reply:
Hmm, some kind of like that I think.

Reply:
Added it. But I am not quite sure how v2alpha API info here is relevant for this section (Connecting to a Public Node).